src/main/java/com/gmail/goosius/siegewar/enums/SiegeStatus.java

```diff
@@ -72,6 +72,35 @@ public String getName() {
 		return this.name;
 	}
 
+	public boolean allowsPlundering() {
+		return attackersWon();
+	}
+
+	public boolean allowsInvading() {
+		return attackersWon();
+	}
+
+	public boolean attackersWon() {
+		switch(this) {
+		case ATTACKER_WIN:
+		case ATTACKER_CLOSE_WIN:
+		case DEFENDER_SURRENDER:
+			return true;
+		default:
+			return false;
+		}
+	}
+
+	public boolean defendersWon() {
+		switch(this) {
+		case DEFENDER_WIN:
+		case ATTACKER_ABANDON:
+			return true;
+		default:
+			return false;
+		}
+	}
+
```

Comment:
- Getting the warchest back is strongly related to the siegestatus, so should presumably be included too e.g.
  - conquest siege & attacker wins .... attacker gets it
  - conquest siege & defebder wins  ......defender gets it

Reply:
Certainly might be a use case for it, yes. I will look into it tomorrow.

Reply:
This did get done.